src/EffectID.h

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+enum class EffectID
```

Comment:
I was also saying they should be sized, probably Int32 (our typedef for int32_t, predates the standard typedefs)